worldedit/manipulations.lua

```diff
@@ -38,6 +38,29 @@ function worldedit.set(pos1, pos2, node_names)
 	return worldedit.volume(pos1, pos2)
 end
 
+--- Sets param2 of a region.
+-- @param pos1
+-- @param pos2
+-- @param param2 Value of param2 to set
+-- @return The number of nodes set.
+function worldedit.set_param2(pos1, pos2, param2)
+	pos1, pos2 = worldedit.sort_pos(pos1, pos2)
+
+	local manip, area = mh.init(pos1, pos2)
+	local param2_data = {}

```

Comment:
The data got via vmanip is probably more than from pos1 to pos2. What happens when param2_data at a specific position is nil? Does the param2 value become set to 0?

Reply:
Sounds reasonable.